src/reducers/articleReducers.js

```diff
@@ -37,44 +37,50 @@ export default function (state = initialState, action) {
         ...state,
         articleData: { ...action.payload },
         actionCalled: true,
-        created: true
+        created: true,
       };
+
     case actionTypes.GET_ALL_ARTICLES:
       return {
         ...state,
         actionCalled: true,
-        articles: action.payload.results,
+        articles: state.articles.concat(action.payload.results),
         nextPage: action.payload.next,
         previousPage: action.payload.previous,
         articleCount: action.payload.count,
       };
+
```

Comment:
Why the black lines?

Reply:
I have removed them
I thought with the blank lines the file looks more readable